anago

```diff
@@ -763,7 +807,7 @@ get_build_candidate () {
 
   # For master branch alpha builds, ensure we've released the previous .0 first
   if [[ -z "$PARENT_BRANCH" && -n ${RELEASE_VERSION[alpha]} ]]; then
-    latest_official=${BASH_REMATCH[1]}.$((${BASH_REMATCH[2]}-1))
+    latest_official="${BASH_REMATCH[1]}.$((${BASH_REMATCH[2]}-1)).0"
```

Comment:
Yes that was a change that slipped in from another PR I was working on.  Thanks!  PTAL.